RELEASE/scripts/autoscend.ash

```diff
@@ -5281,19 +5281,19 @@ boolean LX_getStarKey()
 		return false;
 	}
 
-	if((item_amount($item[Star]) >= 8) && (item_amount($item[Line]) >= 7))
+	//softcore stop adventuring in hole in the sky if you have everything but the star chart. The chart will be pulled in auto_tower.ash
+	if((item_amount($item[Star]) >= 8) && (item_amount($item[Line]) >= 7) && !in_hardcore())
 	{
-		if(!in_hardcore())
-		{
-			set_property("auto_getStarKey", false);
-			return false;
-		}
+		set_property("auto_getStarKey", false);
+		return false;
 	}
```

Comment:
I would move the star chart pull here (and remove it from auto_tower.ash). A lot of this logic has existed since this was cc_ascend so the approach to things is very scattershot in places. However we can and should consolidate things.

Reply:
done at commit a88f3de